app/Resources/SonataAdminBundle/views/Core/user_block.html.twig

```diff
@@ -0,0 +1,7 @@
+{% block user_block %}
+    <br/>
+    <a href="{{ path('sonata_admin_dashboard') }}">&nbsp;&nbsp;Main page</a>
+    <br/><br/>
+    <a href="{{ path('nelmio_api_doc_index') }}">&nbsp;&nbsp;Logout</a>
```

Comment:
It's not a logout, you need to rename anchor of this link, or make a ticket for update this link